### PR TITLE
Reduces bee's damage

### PR DESCRIPTION
**Pull request by Alberyk**
Bees could kill people with a single sting, this will reduce greatly their damage.

### CODE REVIEW

File: code/game/objects/items/bees_items.dm [hunk 1]
```diff
@@ -47,7 +47,7 @@
 /obj/item/weapon/book/manual/hydroponics_beekeeping
 	name = "The Ins and Outs of Apiculture - A Precise Art"
 	icon_state ="bookHydroponicsBees"
-	author = "Beekeeper Nanako"
+	author = "Beekeeper Alberyk"
```
Comment (Lohikar) [inline code/game/objects/items/bees_items.dm]: Yeah, I thought you were meming. Change this to a random real name.

File: code/game/objects/items/bees_items.dm [hunk 2]
```diff
@@ -47,7 +47,7 @@
 /obj/item/weapon/book/manual/hydroponics_beekeeping
 	name = "The Ins and Outs of Apiculture - A Precise Art"
 	icon_state ="bookHydroponicsBees"
-	author = "Beekeeper Nanako"
```
Comment (Belsima) [inline code/game/objects/items/bees_items.dm]: why is nanako die for the most generic name ever? just curious

**Comment by Belsima** [comment]
i mean idk ive seen a single sting kill irl

File: code/game/objects/items/bees_items.dm (hunk 2 above)
Comment (Lohikar) [reply code/game/objects/items/bees_items.dm]: Nanako is the name of a previous coder, and the same one who made this bee change.

Comment (Belsima) [reply code/game/objects/items/bees_items.dm]: Ah, alright, thanks. Could we get anything but Dave, though? I've seen so many Daves and Bills.

Comment (Lohikar) [reply code/game/objects/items/bees_items.dm]: Reason Dave was used is that that was the old name.

Comment (Belsima) [reply code/game/objects/items/bees_items.dm]: I think that's an example of it being used all the time, but fair enough there.

**Pull request merged**